Example/StompClientLib.swift

```diff
@@ -90,16 +90,12 @@ public class StompClientLib: NSObject, SRWebSocketDelegate {
         }
     }
     
-    public func openSocketWithURLRequest(request: NSURLRequest, delegate: StompClientLibDelegate) {
```

Comment:
Actually without header still would be useful. It should stay with overload methods.

Reply:
Yup, you right, but I keep the same function with default connectionHeaders properties with null value with one function for the same logic, don't need override. With default connectionHeaders value in function define, you can use with without header or not, like 
```
Without headers with defaults value is null:
socketClient.openSocketWithURLRequest(request: request, delegate: delegate)

With headers:
socketClient.openSocketWithURLRequest(request: request, delegate: delegate, connectionHeaders: headers)
```

Reply:
Awesome answer :) Have you tested it? Does it work with your way? If it is, I will merge it and publish a new version 🎉

Reply:
Yup, I have tested it with my project.

Reply:
Nice job :) I'm gonna test it and release it soon